tutorials/01_04_coordinate_systems.ipynb

```diff
@@ -0,0 +1,384 @@
+{
```

Comment:
<p>I'd use an RST heading link to the WeldxFile tutorial instead of pointing to the raw (like JSON code) notebook code on Github.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/793/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
The problem with RTS links is that they don't work in the notebooks themselves. The current links work for both, online documentation and notebooks.